client/cmd/login.go

```diff
@@ -43,6 +44,8 @@ var loginCmd = &cobra.Command{
 				AdminURL:      adminURL,
 				ConfigPath:    configPath,
 				PreSharedKey:  &preSharedKey,
+				WgIface:       wgIface,
```

Comment:
We don't need to add the ports in the login command as it doesn't start the daemon. Keeping on up should be enough and prevents confusion.